<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteUploadsDialog</class>
 <widget class="QDialog" name="DeleteUploadsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>This will delete all your uploaded music.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="addUploadsCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This will attempt to add each album or song to your library from YouTube Music's online catalog before deleting it from your uploads. If a match could not be found, the album or song will remain in your uploads.</string>
   </property>
   <property name="text">
    <string>Add uploads to library first</string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreCutoffLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>A value closer to 100 will be more strict regarding matches when searching YTMusic for the song/album. A value of 100 will basically only add exact matches. If you find that not many matches are being found, try lowering this value, but you may end up with albums in your library that are not exact matches. 85 is recommended to start out with.</string>
   </property>
   <property name="text">
    <string>Match score cutoff:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="scoreCutoffInput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>A value closer to 100 will be more strict regarding matches when searching YTMusic for the song/album. A value of 100 will basically only add exact matches. If you find that not many matches are being found, try lowering this value, but you may end up with albums in your library that are not exact matches. 85 is recommended to start out with.</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>85</number>
   </property>
  </widget>
  <widget class="QLabel" name="hoverOverLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Hover over these options for more info)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeleteUploadsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeleteUploadsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
